third_party/move/move-compiler/tests/move_check/parser/duplicate_attributes.exp

```diff
@@ -1,3 +1,15 @@
+error[E02015]: invalid attribute
```

Comment:
Expected different errors here I think

Reply:
I copied the same check to the /skip_attribute_checks/ directory where you can see the old result.